### PR TITLE
Dont spawn enderdragon

### PR DESCRIPTION
**Pull request by DoctaEnkoda**

### CODE REVIEW

**Comment by DoctaEnkoda** [comment]
Partial Fix #361 

Structure Exit Portal always generate

File: src/main/java/com/iridium/iridiumskyblock/api/IridiumSkyblockAPI.java [hunk 1]
```diff
@@ -275,4 +275,16 @@ public boolean isIslandWorld(World world) {
         return Objects.equals(getWorld(), world) || Objects.equals(getNetherWorld(), world) || Objects.equals(getEndWorld(), world);
     }
 
+    public boolean isIslandOverWorld(World world) {
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/api/IridiumSkyblockAPI.java:278]: Just link these to IslandManager

**Review by MertUnverdi — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/listeners/EntitySpawnListener.java [hunk 2]
```diff
@@ -22,4 +24,14 @@ public void onEntitySpawn(EntitySpawnEvent event) {
         });
     }
 
+    @EventHandler
+    public void onCreatureSpawn(CreatureSpawnEvent event) {
+        if (event.isCancelled()) return;
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/listeners/EntitySpawnListener.java:29]: use ignoreCancelled instead this

Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntitySpawnListener.java:29]: Seems unnecessary a bit

**Review by MertUnverdi — APPROVED** [review]

**Pull request merged**